modules/test/playwright/tests/batch-planner/dependencies/c_test.csv

```diff
@@ -0,0 +1,3 @@
+dateModified,dateCreated,externalReferenceCode,id,name,scopeKey,testDateField,testDateTimeField,testDecimalField,testIntegerField,testLongInteger,testLongTextField,testPrecisionDecimalField,testRichTextField
```

Comment:
maybe for this file the name could be c_test-AvailableHeaders or something similar? For other files it's lear to me what it has or hasn't even without looking at the content :)

Reply:
Yesterday @4lejandrito proposed some changes on file names too. Are you ok with them? :smile:

Reply:
oh, will check the file names then, I thought only test names were changed :D